website/_dogfooding/dogfooding.config.js

```diff
@@ -24,7 +24,7 @@ const dogfoodingPluginInstances = [
   ],
 
   [
-    '@docusaurus/plugin-content-blog',
```

Comment:
going to dogfood other forms of plugin names here, it's the purpose of dogfooding to have a variety of settings ;)